src/store/index.ts

```diff
@@ -1,25 +1,26 @@
 import { defineStore } from 'pinia';
 import { getAllMoviesData, getMovieGenresData, getSeancesData } from '@/services/api/data.js';
+import { GenreData, MovieData, SeanceData, ScreeningData } from '@/types/data';
 import router from '@/router';
 
 export const useMainStore = defineStore('main', {
   state() {
     return {
-      error: null,
-      errorDisplayTime: 4,
+      error: null as Error,
+      errorDisplayTime: 4 as number,
       areMoviesLoading: false,
       areGenresLoading: false,
       areSeancesLoading: false,
       areScreeningsLoading: false,
-      allMovies: [],
-      allGenres: [],
-      storedSeances: [],
-      storedScreenings: [],
+      allMovies: [] as Array<MovieData>,
+      allGenres: [] as Array<GenreData>,
+      storedSeances: [] as Array<SeanceData>,
+      storedScreenings: [] as Array<ScreeningData>,
```

Comment:
You might like to know that you can also use the shorthand for this if you like to 😄 
```suggestion
      allMovies: [] as MovieData[],
      allGenres: [] as GenreData[],
      storedSeances: [] as SeanceData[],
      storedScreenings: [] as ScreeningData[],
```